pesacheck_meedan_bridge/py/check_api.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+import requests
+import settings
+
+
+def create_mutation_query(
+    media_type=None,
+    channel=None,
+    set_tags=None,
+    set_status=None,
+    set_claim_description=None,
+    title=None,
+    summary=None,
+    url=None,
+    language=None,
+    publish_report=False,
+):
+    mutation_query = f"""
+      mutation create {{
+        createProjectMedia(input: {{
+          media_type: "{media_type or "Blank"}",
```

Comment:
Why not set `"Blank"` as the default value for `media_type` in the method params? Similar to all other params.